### PR TITLE
Adding Pagination to blog

### PR DESCRIPTION
**Pull request by julien51**
# Description

I was looking for old posts and could not find them, so I added pagination ;)
Since we only have 20 posts in the current setup that's just 2 pages. I changed locally to 3 posts per page to make sure pagination worked as expected. (shows first only when on any other page, last on another page, and previous/next when applicable)

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

**Comment by cnasc** [comment]
There seems to be a lot of unrelated changes in here, you might need to rebase?

**Comment by julien51** [comment]
hum, strange. indeed.

File: unlock-protocol.com/next.config.js [hunk 1]
```diff
@@ -76,7 +76,6 @@ module.exports = withTypescript({
       '/jobs': { page: '/jobs' },
       '/terms': { page: '/terms' },
       '/privacy': { page: '/privacy' },
-      '/post': { page: '/post' },
```
Comment (julien51) [inline unlock-protocol.com/next.config.js]: This was not used...

File: unlock-protocol.com/src/pages/post.js [hunk 2]
```diff
@@ -60,7 +60,8 @@ Post.propTypes = {
 }
 
 Post.getInitialProps = async context => {
-  return await preparePostProps(context)
+  const { slug } = context.query
+  return await preparePostProps(slug)
```
Comment (julien51) [inline unlock-protocol.com/src/pages/post.js]: I think the `context` object is specific to next so I'd rather not pass it down to `preparePostProps`

**Comment by julien51** [comment]
Weird that the paywall integration tests are failing here :/

File: unlock-protocol.com/src/pages/post.js (hunk 2 above)
Comment (benwerd) [reply unlock-protocol.com/src/pages/post.js]: :+1:

File: unlock-protocol.com/next.config.js (hunk 1 above)
Comment (benwerd) [reply unlock-protocol.com/next.config.js]: Good catch

**Pull request merged**